common/qconnector/rabbitmq_connector.py

```diff
@@ -8,14 +8,15 @@
 
 
 class RabbitMQConnector(QConnector):
-    def __init__(self, rabbitmq_url, rabbitmq_exchange, queue_name=None, binding_keys=None):
+    def __init__(self, rabbitmq_url, rabbitmq_exchange, queue_name=None, binding_keys=None, use_compression=True):
```

Comment:
Named variable as `use_compression` to later use the same variable not no decompress pgoutput messages